Quantum-Control-Applications/Superconducting/Single-Fixed-Transmon/configuration.py

```diff
@@ -155,13 +155,11 @@ def IQ_imbalance(g, phi):
 
 opt_weights = False
 if opt_weights:
-    from qualang_tools.config.integration_weights_tools import convert_integration_weights
-
     weights = np.load("optimal_weights.npz")
-    opt_weights_real = convert_integration_weights(weights["weights_real"])
-    opt_weights_minus_imag = convert_integration_weights(weights["weights_minus_imag"])
-    opt_weights_imag = convert_integration_weights(weights["weights_imag"])
-    opt_weights_minus_real = convert_integration_weights(weights["weights_minus_real"])
+    opt_weights_real = weights["weights_real"]
+    opt_weights_minus_imag = weights["weights_minus_imag"]
+    opt_weights_imag = weights["weights_imag"]
+    opt_weights_minus_real = weights["weights_minus_real"]
 else:
     opt_weights_real = [(1.0, readout_len)]
```

Comment:
Hey Kevin, I know this isn't in the scope of your solution, but are the weights in the un-optimized `else` section wrong? Shouldn't the first index (weight) of each tuple be 1, 0 or -1?

Reply:
Hi Dean, yes its wrong an ill defined , I just fixed it